service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -210,8 +212,13 @@ private GA4GHPassport buildPassport(Jwt passportJwt) {
   }
 
   private GA4GHVisa buildVisa(Jwt visaJwt) {
+    JSONObject visaClaims = getJwtClaim(visaJwt, GA4GH_VISA_V1_CLAIM);
```

Comment:
I think this will work:
```suggestion
    var visaClaims = getJwtClaim<JSONObject>(visaJwt, GA4GH_VISA_V1_CLAIM);
```

Reply:
nope, but this works
`var visaClaims = this.<JSONObject>getJwtClaim(visaJwt, GA4GH_VISA_V1_CLAIM);`
but I don't really like it